src/config/lang/english.go

```diff
@@ -515,3 +515,8 @@ var (
 	ErrUnableToCheckArch = errors.New("unable to get the configured cluster's architecture")
 	ErrInterrupt         = errors.New("Failed due to interrupt")
 )
+
+// Collection of reusable warn messages.
+var (
+	WarnSGetDeprecation = "Using sget to download resources is being deprecated and will removed in the v0.29.0 release of Zarf. Please use {@JPERRY} instead."
```

Comment:
Please use `{@JPERRY}`

Reply:
also v 30

Reply:
Also also, 31 may be better to say to give a full 2 releases... That was what I originally put actually